static/app/views/starfish/views/spans/useModuleFilters.ts

```diff
@@ -0,0 +1,22 @@
+import pick from 'lodash/pick';
+
+import {useLocation} from 'sentry/utils/useLocation';
+import {SpanMetricsFields} from 'sentry/views/starfish/types';
+
+type Query = {
+  'span.action': string;
+  'span.domain': string;
+  'span.group': string;
+  'span.op': string;
```

Comment:
l: should we key by the constant names here? `SpanMetricsFields.SPAN_ACTION`

Reply:
Done 👌🏻